### ISSUE TITLE
[axe-core warning] - Ensure <img> elements have alternate text or a role of none or presentation

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#2389]
# Feature Request

Need to update our test examples to ensure `<img>` elements have alternate text or a role of none or presentation.

## Description
<details>
 <summary>WDIO Axe Error Report</summary>

```
9) Image Default is accessible:
expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"87ec563ff16c288f58b0747d553762d6.jpg\" alt=\" \" class=\"Image-module__image___sXcA7 default\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        "img"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
running chrome
AssertionError: expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"87ec563ff16c288f58b0747d553762d6.jpg\" alt=\" \" class=\"Image-module__image___sXcA7 default\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        "img"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
    at Context.<anonymous> (/Users/bj031910/repos/terra-core/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessiblity.js:13:54)
    at new Promise (<anonymous>)
    at new F (/Users/bj031910/repos/terra-core/node_modules/core-js/library/modules/_export.js:36:28)

10) Image Loading is accessible:
expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"87ec563ff16c288f58b0747d553762d6.jpg\" alt=\" \" class=\"Image-module__image___sXcA7 default\" id=\"loadedImage\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        "#loadedImage"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
running chrome
AssertionError: expected no accessibility violations but got:
        {
  "description": "Ensures <img> elements have alternate text or a role of none or presentation",
  "help": "Images must have alternate text",
  "helpUrl": "https://dequeuniversity.com/rules/axe/3.2/image-alt?application=axeAPI",
  "id": "image-alt",
  "impact": "critical",
  "nodes": [
    {
      "all": [],
      "any": [],
      "failureSummary": "Fix all of the following:\n  Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
      "html": "<img src=\"87ec563ff16c288f58b0747d553762d6.jpg\" alt=\" \" class=\"Image-module__image___sXcA7 default\" id=\"loadedImage\">",
      "impact": "critical",
      "none": [
        {
          "data": null,
          "id": "alt-space-value",
          "impact": "critical",
          "message": "Element has an alt attribute containing only a space character, which is not ignored by all screen readers",
          "relatedNodes": []
        }
      ],
      "target": [
        "#loadedImage"
      ]
    }
  ],
  "tags": [
    "cat.text-alternatives",
    "wcag2a",
    "wcag111",
    "section508",
    "section508.22.a"
  ]
}
    at Context.<anonymous> (/Users/bj031910/repos/terra-core/node_modules/terra-toolkit/lib/wdio/services/TerraCommands/accessiblity.js:13:54)
    at new Promise (<anonymous>)
    at new F (/Users/bj031910/repos/terra-core/node_modules/core-js/library/modules/_export.js:36:28)
```
</details>
